packages/renderer/src/lib/preferences/PreferencesRenderingItemFormat.spec.ts

```diff
@@ -476,3 +476,27 @@ test('Expect boolean record to be updated from checked to not checked', async ()
   // checkbox should not be checked anymore
   await vi.waitFor(() => expect(checkbox).not.toBeChecked());
 });
+
+test('Expect a password input when record is type string and format is password', async () => {
+  const record: IConfigurationPropertyRecordedSchema = {
+    id: 'record',
+    title: 'record',
+    parentId: 'parent.record',
+    description: 'record-description',
+    format: 'password',
+    type: 'string',
+  };
+  await awaitRender(record, {});
+  const passwordInput = screen.getByLabelText('password input-standard-record');
```

Comment:
Suggestion:
  const { getByLabelText } = await awaitRender(record, {});
  const passwordInput = getByLabelText('password input-standard-record');

(and change return of awaitRender)

Reply:
it's a copy/paste from the other tests